src/app/services/auth.service.ts

```diff
@@ -0,0 +1,61 @@
+import { Injectable, inject } from '@angular/core';
+import { HttpClient } from '@angular/common/http';
+import { Observable, tap } from 'rxjs';
+import { RegisterResponse } from '../models/register-response.model';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class AuthService {
+  private apiUrl = 'http://localhost:3000/auth/login';
```

Comment:
Move all this URL configurations to an environment file: [https://angular.dev/tools/cli/environments#configure-environment-specific-defaults](url)